setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="threat_intel",
-    version='0.1.30',
+    version='1.0',
```

Comment:
We usually use full semver, so it would be `1.0.0`.
Also, we have been incrementing the patch version for some pretty significant changes, maybe it's just enough to have minor bump in this case, so `0.2.0`, no hard preferences though.

Reply:
cool, fixed